src/v2/Apps/Partner/Components/PartnerHeader/PartnerHeaderAddress.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import styled from "styled-components"
+import { uniq } from "lodash"
+import { PartnerHeader_partner } from "v2/__generated__/PartnerHeader_partner.graphql"
+
+export const TextWithNoWrap = styled.span`
+  white-space: nowrap;
+`
+export const PartnerHeaderAddress: React.FC<
+  PartnerHeader_partner["locations"]
+> = ({ edges }) => {
+  return (
+    <>
+      {uniq(edges.map(edge => edge.node.city))
```

Comment:
Will each `edge.node` definitely have a `city` present?

Reply:
Not sure, but looks like city field is required in CMS.
![image](https://user-images.githubusercontent.com/79979820/112043310-926e8980-8b59-11eb-928c-9e7ae6377217.png)